app/_config/config.yml

```diff
@@ -3,6 +3,4 @@ Name: app
 ---
 SilverStripe\Admin\LeftAndMain:
   extra_requirements_javascript:
-    - app/javascript/googleanalytics.js
-SilverStripe\Security\Security:
-  default_login_dest: admin/pages
```

Comment:
We need to keep this. This is what brings you straight into the CMS after logging in. e.g
1. Click "Log in to try the CMS" from the home page
2. Log in
3. You should be taken to `/admin/pages` but instead you are just stuck looking at a screen that tells you you're logged in

Reply:
Added back in